.travis.yml

```diff
@@ -1,6 +1,19 @@
 language: c
 
-script: printenv && uname -a && ls -l && /bin/sh -x ./bin/setup.sh && cd build && ../configure --enable-warnings && make && sudo make install && make test
+os:
+- linux
+- freebsd
+
+before_install:
+  - export CC=cc CXX=c++
```

Comment:
This is required because:
- GNU autoconf and TravisCI set `CC=gcc` by default
- Linux build is broken with Clang
- FreeBSD build is broken with GCC

Reply:
Not necessary after https://github.com/travis-ci/travis-build/pull/1915